dm/master/server.go

```diff
@@ -369,7 +369,7 @@ func (s *Server) UpdateTask(ctx context.Context, req *pb.UpdateTaskRequest) (*pb
 	if err != nil {
 		return &pb.UpdateTaskResponse{
 			Result: false,
-			Msg:    errors.ErrorStack(err),
```

Comment:
why call `Cause()` here?

Reply:
not necessary. only to drop the annotated message to make check result clearer.

Reply:
ok